Description: Renamed launcher file to Play2048
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 2048 (1.7-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Raphael Gontijo Lopes <raphaelgontijolopes@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- 2048-1.7.orig/Play2048.desktop
+++ 2048-1.7/Play2048.desktop
@@ -3,9 +3,9 @@ Version=1.0
 Type=Application
 Terminal=true
 Icon[en_US]=gnome-panel-launcher
-Exec=/usr/share/doc/2048/2048
-Name[en_US]=2048
+Exec=/usr/share/2048/2048
+Name[en_US]=Play2048
 Comment[en_US]=Play 2048 in Linux's terminal.
 Name=2048
 Comment=Play 2048 in Linux's terminal.
-Icon=/usr/share/doc/2048/2048icon.png
+Icon=/usr/share/2048/2048icon.png
